opal/corelib/process/status.rb

```diff
@@ -1,21 +1,75 @@
 module ::Process
   class Status
+    def self.wait(pid = -1, flags = 0)
+      # Like Process.wait, but returns a Process::Status object (instead of an integer pid or nil)
+      _pid, status = ::Process.wait2(pid, flag)
+      status
+    end
+
     def initialize(status, pid)
       @status, @pid = status, pid
     end
 
-    def exitstatus
-      @status
+    def ==(other)
+      # Returns whether the value of to_i == other
+      to_i == other
     end
 
-    attr_reader :pid
+    # Returns true if the process generated a coredump when it terminated, false if not.
+    def coredump?
+      __not_implemented__
```

Comment:
Perhaps we should do some helper for not implemented functions, like:

```ruby
Opal::Raw.not_implemented :coredump?
```

What this helper could do as well, is to make `respond_to?` return `false` on those methods as it does in CRuby.

Reply:
I added a simple check for a alias to __not_implemented__ to #respond_to? in the latest commit [ae6127a](https://github.com/opal/opal/pull/2746/commits/ae6127ab8d22d21be5df9d6b5a2b0bd8c4de1895). This allows for overwriting the alias and #respond_to? will automatically be doing the right thing.

Reply:
However, what is __not_implemented__ clearly depends on drivers too, need to think about that a bit, how i can bring those 2 together